tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1218,21 +1188,27 @@ module.exports = {
 
   //note to CHW
   getNoteToCHW: () => {
-    return element(by.name('/delivery/group_note/g_chw_sms')).getText();
+    return element(by.css('textarea')).getAttribute('value');
   },
 
   //summary page
   getOutcomeText: () => {
     return element(by.css('[data-value=" /delivery/group_delivery_summary/display_delivery_outcome "]'))
-      .getText();
+      .getInnerHtml();
```

Comment:
This is another weird one, where `getText()` returns a null but `getInnerHtml()` returns the expected text:
![image](https://user-images.githubusercontent.com/6979995/27269439-1fbd977e-550b-11e7-8f7f-f6d87887680e.png)